src/components/card.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./card.css";
+import { SlLogin } from "react-icons/sl";
+import Stack from "react-bootstrap/Stack";
+
+function Card({ title, content, label }) {
+  const getArrowDirection = () => {
+    return content >= 150 ? "up" : "down";
+  };
+
+  const getArrowColor = () => {
+    return getArrowDirection() === "up" ? "#009E0F" : "#CC0000";
+  };
+
+  return (
+    <Stack gap={3}>
```

Comment:
You are making good progress! Just some things you need to consider when building re-usable components.

A component usually just do one thing only. For example, this is a card component, so it should only contain anything about card.

Here the `Stack` is a utility to set the layout of multiple `Card` although we only have one for now.  Because of that, this should not be part of `Card` component. The `Card` component should only include elements about the Card itself. I think you can move the `Stack` part in the `Home` for now where you implement the card itself